bin/next-start

```diff
@@ -38,7 +39,14 @@ if (argv.help) {
 const dir = resolve(argv._[0] || '.')
 
 const srv = new Server({ dir })
-srv.start(argv.port)
+
+exists(resolve(process.cwd(), '.next')).then(exists => {
+  if (!exists) {
+    console.error(`> Error! The application should be compiled with \`next build\` first.`)
```

Comment:
How about a message like this:

Could not found the '.next' directory! Try building your app with 'next build' before starting the server.

cc @rauchg